src/Libraries/UnitsUI/UnitsUI.cs

```diff
@@ -227,7 +227,7 @@ public LengthFromString():base()
         {
             Measure = Length.FromDouble(0.0, LengthUnit.FractionalFoot);
 
-            OutPorts.Add(new PortModel(PortType.Output, this, new PortData(Resources.LengthFromStringPortDataLengthToolTip, Resources.LengthFromStringPortDataLengthToolTip)));
+            OutPorts.Add(new PortModel(PortType.Output, this, new PortData("feet", Resources.LengthFromStringPortDataLengthToolTip)));
```

Comment:
hm, `feet` should be in Resources. Why are we removing the `Resources.LengthFromStringPortDataLengthToolTip`?

Reply:
ah its using `LengthFromStringPortDataLengthToolTip` as the name and the tooltip. Anyways the name of the port should be in a resource file as well

Reply:
Tooltips are only used on descriptions not on names in other ports.